backend/src/test/java/com/votogether/domain/post/controller/PostControllerTest.java

```diff
@@ -400,4 +400,40 @@ void postClosedEarly() {
         assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
     }
 
+    @Test
+    @DisplayName("회원본인이 작성한 게시글 목록을 조회한다.")
+    void getPostsByWriter() {
+        // given
+        PostBody postBody = PostBody.builder()
+                .title("title")
+                .content("content")
+                .build();
+
+        Post post = Post.builder()
+                .writer(MALE_30.get())
+                .postBody(postBody)
+                .deadline(LocalDateTime.now().plusDays(3L).truncatedTo(ChronoUnit.MINUTES))
+                .build();
+
+        PostResponse postResponse = PostResponse.of(post, MALE_30.get());
+
+        given(postService.findPostsByWriter(anyInt(), any(), any(), any(Member.class)))
+                .willReturn(List.of(postResponse));
+
+        // when
+        List<PostResponse> result = RestAssuredMockMvc.given().log().all()
+                .param("page", 0)
+                .param("postClosingType", PostClosingType.PROGRESS)
+                .param("postSortType", PostSortType.LATEST)
+                .when().get("/posts/me")
```

Comment:
`P3`
```suggestion
        List<PostResponse> result = RestAssuredMockMvc
                .given().log().all()
                .param("page", 0)
                .param("postClosingType", PostClosingType.PROGRESS)
                .param("postSortType", PostSortType.LATEST)
                .when().get("/posts/me")
```
given, when, then을 한번에 보여주기에는 이렇게 작성하는 것이 가독성이 더 좋다고 생각해요!
루쿠는 어떻게 생각하시나요?

Reply:
좋아요~ 수정하였습니다!